data/hai/hai reveal 2 start.txt

```diff
@@ -0,0 +1,368 @@
+	on offer
+		"Hai Rescue: Ooonem (offer count)" ++
+		conversation
+			`In the shadows, you see a short, hooded, woman. There is a lot of hair sticking out from her hood. She might be a Hai. Do you want to talk to her?`
+			choice
+				`	(Talk to her.)`
+				`	(Come back later.)`
+					defer
+			`	As you approach, she pulls a laser rifle out of her cloak, points it at you, and says, "No! I'm not going back to them!"`
+			`	She sees your confusion and calms down, "You're not one of them. What do you want with me?"`
+			choice
+				`	"I was wondering why a Hai is on a pirate planet."`
+					goto hai
+				`	"Not one of whom?"`
+					goto who
+				`	"Sorry to bother you. I'll leave you to your business."`
+					goto decline
+			label hai
+			`	"I was going to meet a businessman on Shroud to arrange a large-scale import deal for my research project. I asked a human merchant to take me there, but I was tricked. The businessman doesn't really exist; the deal was faked. The merchant locked me in a cell and brought me here instead. When he opened the cell, I managed to grab his rifle, shoot him, and hide. It's amazing how slowly humans move."`
+				goto "more questions"
+			label who
+			`	"There are some men looking for me. I've managed to escape them so far; they're not as quick as a Hai. I think they're friends of the merchant who locked me up."`
+				goto "more questions"
+			label pay
+			`	"I have no money but my family on <planet> can pay."`
+				goto "more questions"
+			label "more questions"
+			`	Can you take me back to <destination>? My family can pay for my return.`
+			choice
+				`	"Yes, I'll take you there."`
+				`	"How much will you pay me?"`
+					goto pay
+				`	"Why are you on a pirate planet?"`
+					goto hai
+				`	"I'm not one of who?"`
+					goto who
+				`	"Sorry, but you'll have to find another ride."`
+					goto decline
+			`	Are you sure? Pirates will chase you. I'd rather not die on your ship.`
+			choice
+				`	"Don't worry. I know how to dodge pirates."`
+				`	"You should find a different ride then."`
+					decline
+			`	"Okay. I guess I should have introduced myself... it's easy to forget social niceties in a place like this. My name is Ooonem, and I'm a researcher from <planet>. Thank you for agreeing to take me home."`
+			`	You notice several men nearby looking intently at Ooonem. The two of you run towards your ship and make it to cover before the men start shooting. Ooonem pulls a grenade out of her cloak and throws it at the men. Gas explodes outward and the men drop dead in seconds.`
+			`	"I had better leave these behind," she says, dropping a few gas grenades on the ground. "I took them off of the pirates a few days ago. They're quite illegal." You notice a label on the grenades "tabun gas: twenty-meter radius." This was a nerve gas grenade, using one of the older varieties of the gas. Cheap and plentiful in pirate space.`
```

Comment:
I'd argue that it would make more sense for the nerve gas here to be sarin instead of tabun. Tabun degrades really quickly over time making it impractical for grenade use, and additionally is both less lethal (taking longer to kill) and harder to turn into a gas. Additionally, some flavor text to the effect of the slavers clutching their throats or shaking on the ground may be a bit truer to life in terms of the effects of nerve gas.